visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.po.ts

```diff
@@ -0,0 +1,30 @@
+export class BlacklistPanelPageObject {
+	async checkExludedListAfterExclusion() {
+		await page.waitForSelector("#excludedList", { visible: true, hidden: false })
+
+		const selector = "#excludedList > md-list-item > div.pattern-text.layout-column > p > bdi"
+
+		const list = await page.$$eval(selector, ls => {
+			return ls.filter(x => {
+				return x.innerHTML === "*html*" || x.innerHTML === "*ts*"
+			})
+		})
+		return list.length === 2
```

Comment:
We could return the list itself. Maybe it will be a bit more dynamic if the number of items will change.

Reply:
I have changed it abit to be more dynamic whereby you can pass in parameters also